app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,7 @@
+class PostsController < ApplicationController
+  def id
+  end
```

Comment:
- [ ] Kindly change the name of this action to "index", instead of "id".

Reply:
Thank you